base/shell/explorer/lang/ro-RO.rc

```diff
@@ -13,16 +13,16 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "&Modifică Data/Ora", ID_SHELL_CMD_ADJUST_DAT
-        MENUITEM "&Personalizare notificări…", ID_SHELL_CMD_CUST_NOTIF
+        MENUITEM "Modifică &Data/Ora", ID_SHELL_CMD_ADJUST_DAT
+        MENUITEM "P&ersonalizare notificări…", ID_SHELL_CMD_CUST_NOTIF
```

Comment:
"D" access key is already assigned two lines below.
"P" (the capital one) is fine as it doesn't have a "descender". See more on [Microsoft's guide on Selecting Access Keys](https://learn.microsoft.com/ro-ro/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design?redirectedfrom=MSDN#selecting-access-keys).